boards/intel/adsp/Kconfig.intel_adsp

```diff
@@ -5,5 +5,8 @@ config BOARD_INTEL_ADSP
 	select SOC_INTEL_CAVS_V25 if BOARD_INTEL_ADSP_CAVS25
 	select SOC_INTEL_CAVS_V25 if BOARD_INTEL_ADSP_CAVS25_TGPH
 	select SOC_INTEL_ACE15_MTPM if BOARD_INTEL_ADSP_ACE15_MTPM
+	select SOC_INTEL_ACE15_MTPM if BOARD_INTEL_ADSP_ACE15_MTPM_SIM
```

Comment:
would this be better as:
```
	select SOC_INTEL_ACE15_MTPM if BOARD_INTEL_ADSP_ACE15_MTPM || 
                                                                    BOARD_INTEL_ADSP_ACE15_MTPM_SIM
```
Don't mind the spacing that was done just to make the examples here easier.  Thought being it'll introduce less risk if the SOC needs to change at some point.